roxie/ccd/ccdcontext.cpp

```diff
@@ -1658,6 +1658,7 @@ class CRoxieContextBase : implements IRoxieAgentContext, implements ICodeContext
         }
         else
         {
+            OwnedSpanScope graphScope = queryActiveSpan()->createInternalSpan(name);
```

Comment:
This seems fine, looking at a sample roxie trace visualization, where would this span reside, is the name good enough, and are there any attributes we should include in the graph span, or are we simply providing the graph's name as reference, and the timing for completeness? 

![image](https://github.com/hpcc-systems/HPCC-Platform/assets/915949/fc69578d-2f8b-4afd-980b-79f79124fe60)

Reply:
It will occur as a child of the main service.  The soapcall activites would be children.

Reply:
We can revisit at a later date to decide if it is worth adding any extra information, but the name and start/duration is what I would be interested in.

Reply:
Let's get it merged and out into the wild, I'll ask for feedback from our users and we can improve later.